src/docs/development/ios-14.md

```diff
@@ -26,15 +26,25 @@ but not debug.
 
 If your iOS 14 app uses text fields, you should build your
 production apps with Flutter 1.20 or the 1.22 beta to
-ensure that clipboard access notifications are not spuriously 
+ensure that clipboard access notifications are not spuriously
 shown when building text fields.
- 
+
+## System font rendering
+
+If your iOS 14 app uses system fonts such as San Francisco (used by default
+by Cupertino widgets), text will not render correctly and will incorrectly
+be shown in a condensed letter spacing due to changes in iOS's font loading
```

Comment:
maybe "text will incorrectly be rendered in a condensed letter spacing due to..."

Reply:
👍